packages/tabs/src/TabTitle.tsx

```diff
@@ -176,9 +173,15 @@ const listTitleStyles = css`
 `;
```

Comment:
Unrelated to this PR, but I don't remember why that `vertical-align: middle;` declaration isn't just inside the block below

Reply:
oh yeah, I'm going to remove it then. It doesn't seem like its needed.